<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplineEditor</class>
 <widget class="QWidget" name="SplineEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SplineEditor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Point #</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Time: 5, value: 5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="linearCheckBox">
        <property name="text">
         <string>Linear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autotangentCheckBox">
        <property name="text">
         <string>Autotangent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpointCheckBox">
        <property name="text">
         <string>Breakpoint</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="xRangeLabel">
        <property name="text">
         <string>xrange</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="yRangeLabel">
        <property name="text">
         <string>yrange</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="splineFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
